media/css/firefox/family-nav.less

```diff
@@ -237,7 +238,7 @@
     }
 
     .primarynav {
-        a {
+        .primary-link {
```

Comment:
Maybe add a <code>cursor: pointer;</code> for :hover? They're no longer links but the spans are still interactive and it feels weird to get the default insert cursor.